op-program/chainconfig/chaincfg_test.go

```diff
@@ -27,16 +27,30 @@ func TestGetCustomRollupConfig_Missing(t *testing.T) {
 
 // TestGetCustomChainConfig tests loading the custom chain configs from test embed FS.
 func TestGetCustomChainConfig(t *testing.T) {
-	config, err := chainConfigByChainID(eth.ChainIDFromUInt64(901), test.TestCustomChainConfigFS)
+	config, err := l2ChainConfigByChainID(eth.ChainIDFromUInt64(901), test.TestCustomChainConfigFS)
 	require.NoError(t, err)
 	require.Equal(t, config.ChainID.Uint64(), uint64(901))
 
-	_, err = chainConfigByChainID(eth.ChainIDFromUInt64(900), test.TestCustomChainConfigFS)
+	_, err = l2ChainConfigByChainID(eth.ChainIDFromUInt64(900), test.TestCustomChainConfigFS)
 	require.Error(t, err)
 }
 
 func TestGetCustomChainConfig_Missing(t *testing.T) {
-	_, err := chainConfigByChainID(eth.ChainIDFromUInt64(11111), test.TestCustomChainConfigFS)
+	_, err := l2ChainConfigByChainID(eth.ChainIDFromUInt64(11111), test.TestCustomChainConfigFS)
+	require.ErrorIs(t, err, ErrMissingChainConfig)
+}
+
+func TestGetCustomL1ChainConfig(t *testing.T) {
+	config, err := l1ChainConfigByChainID(eth.ChainIDFromUInt64(900), test.TestCustomChainConfigFS)
+	require.NoError(t, err)
+	require.Equal(t, config.ChainID.Uint64(), uint64(900))
+
+	_, err = l1ChainConfigByChainID(eth.ChainIDFromUInt64(901), test.TestCustomChainConfigFS)
+	require.Error(t, err)
```

Comment:
(nit) Should this check be moved to `TestGetCustomL1ChainConfig_Missing`?:
```suggestion
	require.ErrorIs(t, err, ErrMissingChainConfig)
```

Reply:
Yes, this is a duplicate, good spot b688379a11120d9d06b0e435983d8126cd2db0da.